src/V3Assert.cpp

```diff
@@ -124,7 +130,7 @@ class AssertVisitor final : public VNVisitor {
     void newPslAssertion(AstNodeCoverOrAssert* nodep, AstNode* failsp) {
         if (m_beginp && nodep->name() == "") nodep->name(m_beginp->name());
 
-        AstNode* const propp = nodep->propp()->unlinkFrBackWithNext();
+        AstNode* propp = nodep->propp()->unlinkFrBackWithNext();
```

Comment:
I don't see where prop is now changed, so perhaps const can remain

Reply:
You're right, I've restored that const (364745f16496aace4d3765eb9b3ed6e5f91c9a7d).